examples/opensora_hpcai/scripts/infer_t5.py

```diff
@@ -38,7 +39,7 @@ def init_env(
     distributed: bool = False,
     max_device_memory: str = None,
     device_target: str = "Ascend",
-    enable_dvm: bool = False,
+    backend: str = "kbk",
```

Comment:
directly to jit_level? not  display kbk, dvm, and ge.

Reply:
done

Reply:
done